app/assets/javascripts/components/GlobalNav.js

```diff
@@ -32,13 +32,177 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities'], function($
     this._setUpMobileInteraction();
     this._setUpDesktopInteraction();
     this._setUpMobileAnimation();
+    this._setUpKeyboardEvents();
 
     $(window).on('resize', utilities.debounce($.proxy(this._setUpMobileAnimation, this), 100));
 
     return this;
   };
 
   /**
+   * Set up keyboard events
+   * Adds enhanced keyboard interation to nav
+   */
+  GlobalNav.prototype._setUpKeyboardEvents = function() {
+    var self = this;
+
+    this.$globalNav.keydown(function(e) {
+      switch (e.which) {
```

Comment:
Do we need this switch statement if both cases are handled identically, and these are the only ones we care about? Would a simple conditional `if (e.which === 40 || e.which === 32)` do the same?